### PR TITLE
API Update API to reflect changes to CLI interaction

### PR DESCRIPTION
**Pull request by GuySartorelli**
Reflects changes in https://github.com/silverstripe/silverstripe-framework/pull/11353

## Issue
- https://github.com/silverstripe/silverstripe-framework/issues/11341

### CODE REVIEW

File: src/Tasks/DeleteAllJobsTask.php [hunk 1]
```diff
@@ -2,52 +2,56 @@
...
  * Use with caution!
  */
 class DeleteAllJobsTask extends BuildTask
 {
-    /**
-     * @inheritdoc
-     * @return string
-     */
-    public function getTitle()
+    protected static string $commandName = 'delete-queued-jobs';
```
Comment (emteknetnz) [inline src/Tasks/DeleteAllJobsTask.php:20]:
```suggestion
    protected static string $commandName = 'DeleteAllJobsTask';
```

File: src/Tasks/Engines/DoormanRunner.php [hunk 2]
```diff
@@ -38,12 +36,12 @@ class DoormanRunner extends BaseRunner implements TaskRunnerEngine
...
 
     /**
-     * Name of the dev task used to run the child process
+     * Name of the command used to run the child process
      *
      * @config
      * @var string
      */
-    private static $child_runner = 'ProcessJobQueueChildTask';
+    private static $child_runner = 'queuedjobs:process-queue-child';
```
Comment (emteknetnz) [inline src/Tasks/Engines/DoormanRunner.php:44]:
```suggestion
    private static $child_runner = 'queuedjobs:ProcessJobQueueChildCommand';
```

File: src/Cli/ProcessJobQueueChildCommand.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+namespace Symbiote\QueuedJobs\Cli;
+
+use Symbiote\QueuedJobs\Services\QueuedJobService;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand('queuedjobs:process-queue-child', hidden: true)]
```
Comment (emteknetnz) [inline src/Cli/ProcessJobQueueChildCommand.php:12]:
```suggestion
#[AsCommand('queuedjobs:ProcessJobQueueChildCommand', hidden: true)]
```

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: src/Tasks/DeleteAllJobsTask.php (hunk 1 above)
Comment (ChloeSartorelli) [reply src/Tasks/DeleteAllJobsTask.php:20]: Discussion about this and other task command names is being held in the main issue.

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**